docs-v2/integrations/all/e-conomic.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: E-conomic
+sidebarTitle: E-conomic
+---
+
+API configuration: [`e-conomic`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (Basic)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [How to generate API credentials](https://www.e-conomic.com/developer/connect)
+-   [API docs](https://www.e-conomic.com/developer/documentation)
+-   [Auth-related docs](https://apis.e-conomic.com/#Accounts..section/Authentication)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+
+## API gotchas
+- E-conomic in Nango uses `Basic` auth mode to access different endpoints. Provide your `YOUR_PRIVATE_TOKEN` as the Username value and `YOUR_AGREEMENT_GRANT_TOKEN` as the Password value.
+- After creating a connection, you can use the Basic credentials as follows;
+```js
+let headers: { [key: string]: string } = {};
+if ('username' in connection.credentials && 'password' in connection.credentials) {
+    headers = {
+        'X-AppSecretToken': connection.credentials.username,
```

Comment:
We can't dynamically add these into the `header` in the providers.yaml? At the very least we can add the content type

Reply:
Yes, currently we can't dynamically add these into the header in the `providers.yaml`. When proxy is constructing headers, it only checks for `connectionConfig`, while the basic credentials are in `credentials`. Thanks for pointing this out. I have added the `content-type` it in the latest [commit](https://github.com/NangoHQ/nango/pull/2557/commits/5b4e53a9cd0c40c1d07b5e68a05c5d35e1706274)